source/messaging/extending-messaging-with-integrations.md

```diff
@@ -1,9 +1,111 @@
+
+#### Office 365 Calendar (E20)
+
+- Two-way integration between Mattermost and Office 365 Calendar, developed by Mattermost.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb" when in a meeting scheduled via Outlook Calendar.
+- Docs: https://mattermost.gitbook.io/plugin-mscalendar
+
+#### Skype for Business
+
+- Start and join voice calls, video calls, and use screen-sharing with your team members. Developed by kosgrz and maintained by Mattermost.
+- Clicking a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
+- Supports a vendor-hosted cloud solution.
+- Source code + docs: https://github.com/mattermost/mattermost-plugin-skype4business
+
+#### Office 365 SSO (E20)
+
+- Configure Mattermost to use your Office 365 credentials and Azure Active Directory account as a single sign-on (SSO) service, developed by Mattermost.
+- Supports Microsoft Active Directory Tenants for team creation, account creation, and SSO.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Docs: https://docs.mattermost.com/deployment/sso-office.html
+
+### CI/CD integrations
+
+There are several ways to connect your CI/CD tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### GitHub
+
+ - Two-way integration between Mattermost and GitHub, developed by Mattermost.
+ - Get reminders on issues and pull requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports GitHub Enterprise, works with SaaS and Enterprise versions of GitHub.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-github 
+
+#### GitLab
+
+ - Two-way integration between Mattermost and GitLab, developed by Romain Maneschi, supported by Mattermost.
+ - Get reminders on issues and merge requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests, and comments.
+ - Supports SaaS and on-prem versions of GitLab.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-gitlab
+
+### Atlassian integrations
+
+There are several ways to connect Atlassian tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### Confluence
+
+- Atlassian Marketplace application for Confluence and Mattermost, developed by codefortynine.
+- Send notifications about page, blogpost, question or comment updates from your Confluence spaces to Mattermost channels.
+- Configure notifications for a specific space, and also user notifications for tasks and mentions.
+- Supports Confluence Cloud, Server and Data Center platforms.
+- Marketplace download + docs: https://marketplace.atlassian.com/apps/1215055/slack-for-confluence
```

Comment:
```suggestion
- Marketplace download + documentation: https://marketplace.atlassian.com/apps/1215055/slack-for-confluence
```

Reply:
Why is this plugin in our docs called "Slack for Confluence"?

Reply:
I have no idea but it is definitely not correct. I have found the correct link.